contrib/core/actions/send_mail/send_mail

```diff
@@ -52,7 +52,7 @@ if [[ -z $trimmed && $SEND_EMPTY_BODY -eq 1 ]] || [[ -n $trimmed ]]; then
 cat <<EOF
 TO: ${TO}
 FROM: ${FROM}
-SUBJECT: ${SUBJECT}
+SUBJECT: =?UTF-8?q?${SUBJECT}?=
```

Comment:
It looks like we might want to base64 encode the subject and change `q` to `B`.

`q` specifies subject is encoded using ``quoted-printable`` encoding which is not the case so it probably won't work correctly.

Reply:
You're right. I've updated the PR here.